glob/CMakeLists.txt

```diff
@@ -125,6 +125,12 @@ new_ec_test(braces_word_choice2 braces.in test.js "^choice=true[ \t\n\r]*$")
 new_ec_test(braces_word_choice3 braces.in test.html "^choice=true[ \t\n\r]*$")
 new_ec_test(braces_word_choice4 braces.in test.pyc "^[ \t\n\r]*$")
 
+# word choice with spaces
+new_ec_test(braces_word_choice5 braces.in test.py "^choice=true[ \t\n\r]*$")
+new_ec_test(braces_word_choice6 braces.in test.js "^choice=true[ \t\n\r]*$")
+new_ec_test(braces_word_choice7 braces.in test.html "^choice=true[ \t\n\r]*$")
+new_ec_test(braces_word_choice8 braces.in test.pyc "^[ \t\n\r]*$")
```

Comment:
Remove line 132, which duplicates line 126.

Reply:
Removed, how can i test that a extension is not matched based on `choice=spaces` might be nice to add a test like that what do you think?

Reply:
Ah!  Sorry for misunderstanding.  Yes, we can test for non-matching.  However, the test filename has to be one that is not matched anywhere in the `.in` file.  How about?---

```diff
-new_ec_test(braces_word_choice8 braces.in test.pyc "^[ \t\n\r]*$")
+new_ec_test(braces_word_choice8 braces.in test.NOMATCH "^[ \t\n\r]*$")
```

Reply:
Ah oke make sense, I thought it might be good idea to test a case that didn't match a specific selector as I had a case that didn't work for me see: https://github.com/editorconfig-checker/editorconfig-checker/issues/198

If you think current test are enough I can agree :)